ci/jenkins/docker-images.ini

```diff
@@ -19,7 +19,7 @@
 [jenkins]
 ci_arm: tlcpackstaging/ci_arm:20230504-142417-4d37a0a0
 ci_cortexm: tlcpackstaging/ci_cortexm:20230504-142417-4d37a0a0
-ci_cpu: tlcpackstaging/ci_cpu:20230504-142417-4d37a0a0
+ci_cpu: tlcpackstaging/ci_cpu:20230513-200357-e54bbc73
```

Comment:
I noticed that all images are pointing to `tlcpackstaging` account rather than `tlcpack` since https://github.com/apache/tvm/pull/14651. Should we fix them?

Reply:
looks the images were uploaded into https://hub.docker.com/u/tlcpack already, we can replace tlcpackstaging with tlcpack for the images with tag 20230504-142417-4d37a0a0.  Probably we can do it in follow-up pr after the cpu image with tag 20230513-200357-e54bbc73 is uploaded into tlcpack

Reply:
cc: @driazati

Reply:
According to https://tvm.apache.org/docs/contribute/ci.html#updating-a-docker-image-tag, the current protocol is to reference `tlcpack` in a PR that updates an image, and CI automatically fetches from the corresponding image from `tlcpackstaging`. The images are automatically uploaded to `tlcpack` after successful merge.